relay-server/src/lib.rs

```diff
@@ -262,6 +261,7 @@ mod http;
 mod metrics_extraction;
 mod middlewares;
 mod service;
+mod services;
```

Comment:
To make naming less awkward here, we might want to move `service` to `services::utils` or something similar.